src/features/auth/userSlice.js

```diff
@@ -0,0 +1,35 @@
+import {createAsyncThunk, createSlice, unwrapResult} from '@reduxjs/toolkit'
+import userApi from '../../apis/userApi'
+
+
+export const processLogin = createAsyncThunk(
+    "user/login",
+    async (user) => {            
+
+        const respone = await userApi.login(user)
+        
+        localStorage.setItem('access_token', respone.data.content.accessToken)
+        return respone.data.content
+
+    }
+)
+
+
+const userSlice = createSlice({
+    name: "user",
+    initialState: {
+        userInfo: {},
+    },
+    extraReducers: {
+        [processLogin.pending]: (state, action) => {
+            state.statusCode = "wating for check login"
```

Comment:
state.statusCode doesn't make sense here.  You need to store pending states when you want to handle loading. That should be handled at Login component